pkg/config/testdata/containers_default.conf

```diff
@@ -121,6 +121,9 @@ default_subnet_pools = [{"base" = "10.89.0.0/16", "size" = 24}, {"base" = "10.90
 
 default_rootless_network_cmd = "pasta"
 
+# firewall driver to be used by default
+firewall_driver = ""
```

Comment:
Here you should add a value different from the empty default, i.e. "none", to ensure that the field is correct read from the conf file.